extensions/orcid.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "ORCID Social Identity Provider",
+    "description": "This [Keycloak](https://www.keycloak.org) plugin makes possible to add [ORCID](https://orcid.org/) as Social Identity Provider.",
+        "cgeorgilakis", "laskasn"
+    ],
```

Comment:
```suggestion
    "description": "Enables <a href=\"https://orcid.org/\">ORCID</a> as an Identity Provider.",
    "maintainers": [
        "cgeorgilakis", "laskasn"
    ],
```

Reply:
Sorry, by mistake. I have corrected it.

Reply:
Could you also change the description? It doesn't allow markdown in the description, and I don't think you need to write "Keycloak plugin" as folks are after all seeing this on a list of Keycloak plugins on the Keycloak website ;)

Reply:
Done.